automation-testing/src/main/java/stepDefinitions/BattlefieldChinaSteps.java

```diff
@@ -20,43 +21,48 @@ public void navigateToTopPage(){
         home.navigate_Top();
     }
 
-    @When("^I select 'PlayStation 4' in platform dropdown$")
-    public void filterGamesListPlatform (){
-        top.selectPlatformOption("PlayStation 4");
+    @When("^I select \"(.*)\" in platform dropdown$")
+    public void filterGamesListPlatform (String platformOpt){
+
+        top.selectPlatformOption(platformOpt);
     }
-    @And("^I select 'Action' in genre dropdown$")
-    public void filterGamesListGenre(){
-        top.selectGenreOption("Acción");
+
+    @And("^I select \"(.*)\" in genre dropdown$")
+    public void filterGamesListGenre(String genreOpt){
+        top.selectGenreOption(genreOpt);
     }
 
-    @And("^I select '2013' in year dropdown$")
-    public void filterGamesListYear(){
-        top.selectYearOption("2013");
+    @And("^I select \"(.*)\" in year dropdown$")
+    public void filterGamesListYear(String yearOpt){
+
+        top.selectYearOption(yearOpt);
     }
-    @And("^I click on the 'Battlefield 4: China Rising' game$")
-    public void navigateToBattlefieldGame(){
-        top.clickBattlefieldGame();
+
+    @And("^I click on the \"(.*)\" game$")
+    public void navigateToBattlefieldGame(String gameLink){
+        top.clickBattlefieldGame(gameLink);
     }
 
-    @And("^I select 'PS3' in ficha tecnica$")
-    public void selectPS3Tab(){
-        game.selectPs3("PS3");
+    @And("^I select \"(.*)\" in ficha tecnica$")
+    public void selectTabFromDataSheet(String option){
+
+        game.selectOptionFromData(option);
     }
 
-    @Then("^I verify that 'Voces' label and 'Español' text are displayed$")
-    public void verifyElementsPresent(){
-        game.verify_voicesLabel();
-        game.verify_voicesValue();
+    @Then("^I verify that \"(.*)\" label and \"(.*)\" text are displayed$")
+    public void verifyElementsPresent(String label , String text){
+        Assert.assertTrue(game.verifyLabelDisplayed(label));
+        Assert.assertTrue(game.verifyValueIsDisplayed(text));
```

Comment:
Assertions should not be in the step definitions. Make the assertions in a method in a page object, and then call that method.